src/main/java/com/google/devtools/build/lib/rules/java/JavaOptions.java

```diff
@@ -580,7 +580,7 @@ private Label getHostJavaBase() {
       if (useJDK10AsHostJavaBase) {
         return Label.parseAbsoluteUnchecked("@bazel_tools//tools/jdk:remote_jdk10");
       } else {
-        return Label.parseAbsoluteUnchecked("@bazel_tools//tools/jdk:remote_jdk");
+        return Label.parseAbsoluteUnchecked("@bazel_tools//tools/jdk:host_jdk");
```

Comment:
Well, I stared an embarrasingly long amount of time before I figured out why replacing the remote JDK with the host one is relevant to this change :)

Reply:
Done now.